Library/Formula/tesseract.rb

```diff
@@ -24,10 +25,21 @@ class Tesseract < Formula
   end
 
   option "all-languages", "Install recognition data for all languages"
+  option "with-training-tools", "Install OCR training tools"
+  option "with-opencl", "Enable OpenCL support"
 
   depends_on "libtiff" => :recommended
   depends_on "leptonica"
 
+  if build.with? "training-tools"
```

Comment:
Ah, ok. It wasn't clear to me from [the Homebrew example formula](https://github.com/Homebrew/homebrew/blob/master/Library/Contributions/example-formula.rb) if there was a better way of doing `depends_on` for options.